src/main.tsx

```diff
@@ -7,13 +7,26 @@ import { RootLayout } from '~/layouts/root-layout'
 import { Root } from '~/routes/root'
 import { DesignSystem } from '~/routes/design-system'
 import { BaseLayout } from './layouts/base-layout'
+import { Landing } from './routes/landing'
+import { Login } from './routes/login'
+import { Greeting } from './routes/greeting'
+import { Survey } from './routes/survey/survey'
```

Comment:
```suggestion
import { BaseLayout } from '~/layouts/base-layout'
import { Landing } from '~/routes/landing'
import { Login } from '~/routes/login'
import { Greeting } from '~/routes/greeting'
import { Survey } from '~/routes/survey/survey'
```

절대 경로 import로 변경해주시면 좋을 것 같아요!